src/Nancy/DefaultNancyBootstrapper.cs

```diff
@@ -277,9 +277,16 @@ protected override sealed INancyModule GetModule(TinyIoCContainer container, Typ
         /// <param name="container">Container instance</param>
         private static void AutoRegister(TinyIoCContainer container, IEnumerable<Func<Assembly, bool>> ignoredAssemblies)
         {
-            var assembly = typeof(NancyEngine).Assembly;
+            var assembly = typeof(NancyEngine).GetTypeInfo().Assembly;
 
+#if !DNX
             container.AutoRegister(AppDomain.CurrentDomain.GetAssemblies().Where(a => !ignoredAssemblies.Any(ia => ia(a))), DuplicateImplementationActions.RegisterMultiple, t => t.Assembly != assembly);
+#else
+            var libraryManager =
+                Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.LibraryManager;
+            var thisAssemblyName = typeof(DefaultNancyBootstrapper).GetTypeInfo().Assembly.GetName().Name;
+            var referencing = libraryManager.GetReferencingLibraries(thisAssemblyName);

```

Comment:
This does not seem to be used? I.e it's not feature equivalent of what is in the `#if !DNX`-section.

All of this should probably just change from using the `AppDomain` to using the `IAssemblyCatalog` which is now on the `NancyBootstrapperBase` so you should just be able to use `this.AssemblyCatalog.GetAssemblies().Where(....)`